recipes/hostile/meta.yaml

```diff
@@ -52,3 +52,7 @@ about:
 extra:
   identifiers:
     - doi:10.1093/bioinformatics/btad728
+  additional-platforms:
```

Comment:
No need of additional platforms for noarch recipe

Reply:
Thanks, may I leave as-is?

Reply:
You could remove it in a future update!